packages/edit-post/src/style.scss

```diff
@@ -3,22 +3,6 @@
 @import "./components/meta-boxes/meta-boxes-area/style.scss";
 @import "./components/welcome-guide/style.scss";
 
-/**
- * Animations
- */
-
-// These keyframes should not be part of the _animations.scss mixins file.
-// Because keyframe animations can't be defined as mixins properly, they are duplicated.
-// Since hey are intended only for the editor, we add them here instead.
```

Comment:
With the changes introduced by this PR, there may be some duplication — but IMO it's a much better compromise than all the potential downsides listed in https://github.com/WordPress/gutenberg/pull/64580#discussion_r1751728630